ostd/src/mm/page/mod.rs

```diff
@@ -83,6 +83,45 @@ impl<M: PageMeta> Page<M> {
         }
     }
 
+    /// Check whether the page described by paddr is allocated.
```

Comment:
Use plain text in comments: "Check whether the page at the physical address is allocated".